generate-release-vars.sh

```diff
@@ -53,10 +53,12 @@ ami_name_al2_arm=$(aws ec2 describe-images --region "$region" --owner amazon --i
 # AL2022
 ami_id_al2022_x86=$(aws ssm get-parameters --region "$region" --names /aws/service/ami-amazon-linux-latest/al2022-ami-minimal-kernel-default-x86_64 --query 'Parameters[0].[Value]' --output text)
 ami_name_al2022_x86=$(aws ec2 describe-images --region "$region" --owner amazon --image-id "$ami_id_al2022_x86" --query 'Images[0].Name' --output text)
+kernel_version_al2022_x86=$(grep -o -e "-kernel-[1-9.]*" <<<"$ami_name_al2022_x86")
```

Comment:
I just found that there are new names for us to get AL2022 and AL2022 ARM AMI IDs with the specified kernel version in SSM. Can we update the script to use the new name as we will prefer to support AMIs with certain kernel version and release AMIs with new kernel version after testing is done?

New name field for SSM:
* AL2022: `/aws/service/ami-amazon-linux-latest/al2022-ami-minimal-kernel-5.15-x86_64`
**ami-0a100548f6e970961** in us-west-2
_^ the same AMI ID as using "/aws/service/ami-amazon-linux-latest/al2022-ami-minimal-kernel-default-x86_64" returned so far_
* AL2022 ARM: `/aws/service/ami-amazon-linux-latest/al2022-ami-minimal-kernel-5.15-arm64`
**ami-0a11fe2d14d48cb2d**  in us-west-2
_^ the same AMI ID as using "/aws/service/ami-amazon-linux-latest/al2022-ami-minimal-kernel-default-arm64" returned so far_

Reply:
> we will prefer to support AMIs with certain kernel version and release AMIs with new kernel version after testing is done

I think, ideally, ECS AMIs want to stay in sync with AL on the "recommended" (or default) kernel version. It does mean that we need to work with AL on qualification, before they start pointing the default image to the ones with newer kernels. 

But either way, I'd prefer to take it as a separate issue, which can have a more profound change (changing source AMIs) than what this PR is intended for (just updating ECS AMI names).